client/js/components/GameView/GameOverlays.vue

```diff
@@ -5,7 +5,7 @@
       v-model="waitingForOpponentToCounter"
       opacity=".6"
     >
-      <h1>{{ waitingForOpponetToCounterMessage }}</h1>
+      <h1>{{ showWaitingForOpponetToCounterMessage }}</h1>
```

Comment:
Added the `show` here to help distinguish between computed values and values mapped from the state. I ran into some confusion using `waitingForOpponentToPlayFromDeck` when trying to make all computed props uniform in name. Since it's already a mapped value, things broke.